### PR TITLE
Create pod

### PR DESCRIPTION
**Pull request by KaneCheshire**
- Created a basic pod with example app to enable us to publish it to ccoapods

### CODE REVIEW

File: Example/Pods/Target Support Files/Pods-TABObserverSet_Example-TABObserverSet_Tests/Pods-TABObserverSet_Example-TABObserverSet_Tests-acknowledgements.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Acknowledgements
+This application makes use of the following third party libraries:
+
+## TABObserverSet
+
+ObserverSet is distributed under a BSD license, as listed below.
+
+
+Copyright (c) 2015, Michael Ash
```
Comment (Neil3079) [inline Example/Pods/Target Support Files/Pods-TABObserverSet_Example-TABObserverSet_Tests/Pods-TABObserverSet_Example-TABObserverSet_Tests-acknowledgements.markdown:9]: Is this right?

File: Example/TABObserverSet/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
+//
+//  ViewController.swift
+//  TABObserverSet
+//
+//  Created by KaneCheshire on 05/02/2017.
+//  Copyright (c) 2017 KaneCheshire. All rights reserved.
+//
+
```
Comment (Neil3079) [inline Example/TABObserverSet/ViewController.swift]: does the example project not do anything?

File: Example/Pods/Target Support Files/Pods-TABObserverSet_Example-TABObserverSet_Tests/Pods-TABObserverSet_Example-TABObserverSet_Tests-acknowledgements.markdown (hunk 1 above)
Comment (KaneCheshire) [reply Example/Pods/Target Support Files/Pods-TABObserverSet_Example-TABObserverSet_Tests/Pods-TABObserverSet_Example-TABObserverSet_Tests-acknowledgements.markdown:9]: Here's where it gets interesting! I have no idea. I was hoping someone would know. If we change  that notice, are we not breaking the terms of the original license?

File: Example/TABObserverSet/ViewController.swift (hunk 2 above)
Comment (KaneCheshire) [reply Example/TABObserverSet/ViewController.swift]: Not yet, but was thinking about putting a few examples in there as a separate PR. Didn't want to clog this one up too much as it's messy anyway

**Review by samdods — APPROVED** [review]

**Pull request merged**